pavelib/paver_tests/test_assets.py

```diff
@@ -38,21 +43,90 @@ def test_compile_sass(self, options):
         self.reset_task_messages()
         call_task('pavelib.assets.compile_sass', options={"system": system, "debug": debug, "force": force})
         expected_messages = []
-        if force:
-            expected_messages.append("rm -rf common/static/css/*.css")
-        expected_messages.append("libsass common/static/sass")
         if "lms" in system:
             if force:
-                expected_messages.append("rm -rf lms/static/css/*.css")
-            expected_messages.append("libsass lms/static/sass")
+                expected_messages.append("rm -rf common/static/css/*.css")
```

Comment:
Is this saying that the common directory will be compiled twice when both systems are specified (which is the default). For performance reasons, it would be better not to change this behavior, and instead have common only compiled once.